srcpkgs/naken_asm/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'naken_asm'
+pkgname=naken_asm
+version=2019.03.17
+revision=1
+wrksrc=${pkgname}-${version//./-}
+build_style=configure
+configure_args="--install-prefix=/usr"
```

Comment:
You would have to add DESTDIR to it. Also pls do test the resulting package locally. Your current template still wouldn't result in any files getting added to the package. Getting everything to work can be done like this, the configure script of `nakem_asm` is a pain to work with e.g. requiring every single CFLAG to be passed as its own `--cflag=flag` parameter, instead of just `--cflag="list of flags"`
```diff
diff --git a/srcpkgs/naken_asm/template b/srcpkgs/naken_asm/template
index 648dc2d1ae..9080db3aec 100644
--- a/srcpkgs/naken_asm/template
+++ b/srcpkgs/naken_asm/template
@@ -4,10 +4,17 @@ version=2019.03.17
 revision=1
 wrksrc=${pkgname}-${version//./-}
 build_style=configure
-configure_args="--install-prefix=/usr"
+make_check_target="tests"
+checkdepends="python"
 short_desc="Multi-architecture lightweight assembler/disassembler"
 maintainer="rc-05 <rc23@email.it>"
 license="GPL-3.0-or-later"
 homepage="http://www.mikekohn.net/micro/naken_asm.php"
 distfiles="http://www.mikekohn.net/downloads/${pkgname}/${pkgname}-${version//./-}.tar.gz"
 checksum=56537597be50284e7f93b3d9e0ab4e111d84ff34ead31cab6e7e41e2d77d749c
+
+do_configure() {
+	./configure --install-prefix=${DESTDIR}/usr --cflags=${CFLAGS// / --cflags=} \
+		--ldflags=${LDFLAGS// / --ldflags=} \
+		--compiler-prefix=${XBPS_CROSS_TRIPLET:+${XBPS_CROSS_TRIPLET}-}
+}
```